src/ape/api/accounts.py

```diff
@@ -64,18 +63,28 @@ def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
         """
 
     @abstractmethod
-    def sign_transaction(self, txn: TransactionAPI) -> Optional[TransactionSignature]:
+    def sign_transaction(self, txn: TransactionAPI, **signer_options) -> Optional[TransactionAPI]:
         """
         Sign a transaction.
 
         Args:
           txn (:class:`~ape.api.transactions.TransactionAPI`): The transaction to sign.
+          **signer_options: Additional kwargs given to the signer to modify the signing operation.
 
         Returns:
-          :class:`~ape.types.signatures.TransactionSignature` (optional): The signed transaction.
+          :class:`~ape.api.transactions.TransactionAPI` (optional): A signed transaction.
+            The `TransactionAPI` returned by this method may not correspond to `txn` given as input,
+            however returning a properly-formatted transaction here is meant to be executed.
+            Returns `None` if the account does not have a transaction it wishes to execute.
+
         """
 
-    def call(self, txn: TransactionAPI, send_everything: bool = False) -> ReceiptAPI:
+    def call(
+        self,
+        txn: TransactionAPI,
+        send_everything: bool = False,
+        **signer_options,
```

Comment:
Thoughts on using `**kwargs` here? Or maybe `signer_options: SignerOptions`?

Reason: It could be risky assigning a specific meaning here to the `**` case since `call()` does a lot.
Note: It makes sense in `sign_transaction()` but in `call()` is where I think it gets gray.